shell/platform/windows/testing/win32_window_test.h

```diff
@@ -0,0 +1,64 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Renamed this from `win32_flutter_window_tests.h` and now is subclassing `Win32Window` instead of the Flutter subclass since it wasn't using any specific features from the subclass, and I needed to access the protected method `GetCurrentDPI`.